pages/member-invitations.js

```diff
@@ -16,6 +16,9 @@ const memberInvitationsPageQuery = gqlV2`
   query MemberInvitationsPage($memberAccount: AccountReferenceInput!) {
     memberInvitations(memberAccount: $memberAccount) {
       id
+      inviter {
```

Comment:
@Betree according to your latest message, we should add `id` there.

Reply:
Added id, slug and type.